rubygem-foreman_ansible/rubygem-foreman_ansible.spec

```diff
@@ -72,14 +78,23 @@ exit 0
 %{gem_instdir}/locale
 %exclude %{gem_cache}
 %{gem_spec}
-%{foreman_bundlerd_dir}/%{gem_name}.rb
+%{foreman_bundlerd_plugin}
+%{foreman_apipie_cache_foreman}
```

Comment:
There should also be foreman_apipie_cache_plugin here (https://github.com/theforeman/foreman-packaging/blob/rpm/develop/gem2rpm/foreman_plugin.spec.erb#L163)